src/main/scala/org/broadinstitute/dsde/agora/server/business/AgoraBusiness.scala

```diff
@@ -129,12 +127,16 @@ class AgoraBusiness {
   }
 
   private def validateDockerImage(task: Task) = {
-    if (task.runtimeAttributes.docker.isDefined) {
-      val dockerImageReference = parseDockerString(task.runtimeAttributes.docker.get)
-      if (dockerImageReference.isDefined) {
-        DockerHubClient.doesDockerImageExist(dockerImageReference.get)
-      }
-    }
+    // Per DSDEEPB-2525, in the interests of expediency, we are disabling docker image validation as we do not support validation of private docker images
```

Comment:
I left it here as a place holder since it seems we'll be adding the functionality back in soon.